clients/cli/src/main.rs

```diff
@@ -0,0 +1,552 @@
+        offset: PodU64::from(RecordData::WRITABLE_START_INDEX as u64),
+        slot: PodU64::from(slot),
+        node_pubkey,
+        reporter,
+        destination,
+        shred_1_merkle_root,
+        shred_1_signature,
+        shred_2_merkle_root,
+        shred_2_signature,
+    };
+    let instructions = duplicate_block_proof_with_sigverify_and_prefund(
+        &proof_account.pubkey(),
+        &instruction_data,
+        Some(&payer.pubkey()),
+        &config.rent,
+    );
+    let transaction = Transaction::new_signed_with_payer(
+        &instructions,
+        Some(&payer.pubkey()),
+        &[payer.as_ref()],
+        config.program_client.get_latest_blockhash().await?,
+    );
+    if let Some(_signature) = process_transaction(config, transaction).await? {
+        let report_account = config.rpc_client.get_account(&pda).await?;
+        let report: &ViolationReport =
+            pod_from_bytes(&report_account.data()[..std::mem::size_of::<ViolationReport>()])?;
+        println_display(
+            config,
+            ViolationReportOutput::from_report(pda, report, vec![]).to_string(),
+        );
+    }
+
+    Ok(true)
+}
+
+/// Close a violation report
+/// If `report_account` is specified we close the report account, otherwise we
+/// close any eligible accounts filtered by `reporter`, `destination`, or
+/// `node_pubkey`
+async fn command_close(
+    config: &Config,
+    command_config: CloseCli,
+    matches: &ArgMatches,
+    wallet_manager: &mut Option<Rc<RemoteWalletManager>>,
+) -> CommandResult {
+    let epoch = config.rpc_client.get_epoch_info().await?.epoch;
+    let report_account =
+        get_pubkey_from_source!(command_config, matches, wallet_manager, report_account)?;
+    let node_pubkey =
+        get_pubkey_from_source!(command_config, matches, wallet_manager, node_pubkey)?;
+    let reporter_pubkey =
+        get_pubkey_from_source!(command_config, matches, wallet_manager, reporter)?;
+    let destination_pubkey =
+        get_pubkey_from_source!(command_config, matches, wallet_manager, destination)?;
+    let payer = config.fee_payer()?;
+
+    let ignore_filter = |account_pubkey: Pubkey, report: &ViolationReport| {
+        report_account
+            .map(|pk| pk != account_pubkey)
+            .unwrap_or_default()
+            || node_pubkey
+                .map(|pk| pk != report.pubkey)
+                .unwrap_or_default()
+            || reporter_pubkey
+                .map(|pk| pk != report.reporter)
+                .unwrap_or_default()
+            || destination_pubkey
+                .map(|pk| pk != report.destination)
+                .unwrap_or_default()
+    };
+    let gpa_config = RpcProgramAccountsConfig {
+        account_config: RpcAccountInfoConfig {
+            encoding: Some(UiAccountEncoding::Base64),
+            commitment: Some(config.rpc_client.commitment()),
+            ..RpcAccountInfoConfig::default()
+        },
+        ..RpcProgramAccountsConfig::default()
```

Comment:
for `close` and `display` we have the option of specifying the memcmp filters to gPA rather than the manual `ignore_filter` above. 
Thought it would be overkill since I expect the number of slashing reports to be very low, but can add it here if you think it's worth it.

Reply:
I think the current approach is fine. As you mentioned, there shouldn't be too many of these accounts, and anyone can close them